HACKING.md

```diff
@@ -230,6 +238,12 @@ For more information on testing, please see the Testing section.
 
 ### Testing
 
+Install the following package(s) to satisfy test dependencies.
+
+```
+sudo apt-get install python3-yamlordereddictloader
```

Comment:
(nitpick) please suggest `apt install` as it gives you a nicer progress bar ;)

Reply:
I will change all the apt-get references in HACKING to use apt instead then.

Reply:
Fine for a followup, it's good to be consistent so thanks for this!